### DISCUSSION TITLE
Automatically create a bunch of fields in a repeater

### DISCUSSION THREAD

**Discussion by immohacker** [Budibase/budibase#10859] [Help] [answered]
Hi, is it possible to create in an reapeter a bunch of paragaphs from fields from an datasource?
I have an Datasource with many fields. To create it manaully it takes time.
So i hope i can create all fields in one run.
So that i must change manually attachments fields 
Because they are no text. But its easier to change two , three fields , as create all fields manually.

Thank you!
Best regards Bonkersdeluxe

**Comment by ConorWebb96** [comment, marked as answer]
Hey @immohacker,

You can use the form block and it will auto generate all table columns for you for a table, this can be changed by change the table.
![Form block component](https://github.com/Budibase/budibase/assets/126772285/2c137723-79d8-4b8f-9251-dc5c29fdd506)

The paragraph component isn't something which is used by forms within Budibase. However, you can use this with the repeater and data provider to loop through fields within a table and output a lot of different information based on the bindings supplied to the paragraph component.

As for the dynamically create form fields this currently can't be done within Budibase and there is an open issue here for it. https://github.com/Budibase/budibase/issues/5658H

Hopefully all of this helps

**Comment by immohacker** [comment]
Hi ConorWebb96
Thats an Idea.
Thank you very much!
Best regards Bonkersdeluxe
